app/api/views/briefs.py

```diff
@@ -768,8 +770,12 @@ def edit_opportunity(brief_id):
         not_found(e.message)
     except UnauthorisedError as e:
         forbidden(e.message)
+    except ValidationError as e:
+        abort(e.message)
     except BriefError as e:
         abort(e.message)
+    except DBError as e:
+        server_error(str(e))
```

Comment:
is it possible to use the exception_logger helper instead of catching it?

Reply:
I've removed this `except` block so the exception logger catches this one.